client/src/client.rs

```diff
@@ -846,6 +846,16 @@ pub trait RpcApi: Sized {
         self.call("generatetoaddress", &[block_num.into(), address.to_string().into()])
     }
 
+    /// Mine a block with a set of ordered transactions immediately to a specified
+    /// address or descriptor (before the RPC call returns)
+    fn generate_block(
+        &self,
+        output: &Address,
+        transactions: &[bitcoin::Txid],
```

Comment:
One of the main use cases for the `generateblock` rpc is to perform a double spend of a tx that is in bitcoind's mempool. In that case a raw transaction is required since bitcoind won't be able to find the tx by txid.